src/components/CustomAnimatedModal.tsx

```diff
@@ -56,7 +57,7 @@ export const CustomAnimatedModal = (props: CustomAnimatedModalProps) => {
                         height: windowHeight * 0.85,
                     }}
                 >
```

Comment:
decoupled this component from the RequestSwapStack, it can be used for any child component passed in